app/controllers/students_controller.rb

```diff
@@ -39,12 +39,33 @@ def sped_referral
     end
   end
 
+  def restricted_notes
+    raise Exceptions::EducatorNotAuthorized unless current_educator.can_view_restricted_notes
+
+    student = Student.find(params[:id])
+    @serialized_data = {
+      current_educator: current_educator,
+      student: serialize_student_for_profile(student),
+      feed: {
+        event_notes: student.event_notes
+          .select {|note| note.is_restricted}
+          .map {|event_note| serialize_event_note(event_note)},
+        deprecated: {
+          interventions: []
+        }
+      },
+      educators_index: Educator.to_index,
+      event_note_types_index: event_note_types_index
+    }
+  end
+
   # post
   def event_note # DEPRECATED. Use EventNotesController#create instead.
     clean_params = params.require(:event_note).permit(*[
       :student_id,
       :event_note_type_id,
-      :text
+      :text,
+      :is_restricted
```

Comment:
I don't think `event_note` is called anymore, and this change would need to be made in `EventNotesController#event_note_params` for saving the value of `is_restricted` to work?